packages/wallet/src/redux/protocols/reducer-helpers.ts

```diff
@@ -97,7 +97,11 @@ export function checkStates(
   states
     .filter(ss => ss.state.turnNum > turnNum)
     .map(ss => {
-      const result = checkAndStore(sharedData, ss);
+      const result = checkAndStore(
+        sharedData,
+        ss,
+        sharedData.bytecodeStorage[ss.state.appDefinition]
```

Comment:
We should probably add a selector to `selector.ts` to get the `byteCode` from `sharedData`.

Reply:
My mistake. Fixing in new commit...